e2e_playwright/st_altair_chart_concatenated_select_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction, wait_for_app_run
+
+
+def test_point_selection_repeat_scatter_chart_displays_dataframe(
+    themed_app: Page, assert_snapshot: ImageCompareFunction
+):
+    chart = themed_app.get_by_test_id("stArrowVegaLiteChart").nth(0)
+    expect(chart).to_be_visible()
+    # Pick Non First chart
+    themed_app.mouse.move(750, 250)
+    themed_app.mouse.down()
+    themed_app.mouse.up()
```

Comment:
nit: This probably triggers a rerun or? I think it would be good to put a `wait_for_app_run(themed_app)` after this to make it more obvious. The expect below is enough since the dataframe only appears after the first selection, but that might be something we want to change.

Reply:
The same might also apply to some of the other tests below.